addenda12_test.go

```diff
@@ -70,14 +70,10 @@ func BenchmarkAddenda12Parse(b *testing.B) {
 func testAddenda12ValidRecordType(t testing.TB) {
 	addenda12 := mockAddenda12()
 	addenda12.recordType = "63"
-	if err := addenda12.Validate(); err != nil {
-		if e, ok := err.(*FieldError); ok {
-			if e.FieldName != "recordType" {
-				t.Errorf("%T: %s", err, err)
-			}
-		} else {
-			t.Errorf("%T: %s", err, err)
-		}
+	err := addenda12.Validate()
+	// TODO: are we not expecting any errors here?
```

Comment:
return the fieldError

	if addenda12.recordType != "7" {
		fieldError("recordType", NewErrRecordType(7), addenda12.recordType)
	}